[Translate] translateset blacklist add in DMs errors

This was caused by attempting to add a user id to translateset blacklist add in DMs.
```py
﻿﻿[2020-06-03 15:20:02] [ERROR] red: ConversionError
Traceback (most recent call last):
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 425, in _actual_conversion
    ret = await instance.convert(ctx, argument)
  File "C:\joker\cogs\CogManager\cogs\translate\converters.py", line 36, in convert
    result = guild.get_channel(int(channel_id))
AttributeError: 'NoneType' object has no attribute 'get_channel'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\bot.py", line 892, in invoke
    await ctx.command.invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 799, in invoke
    await super().invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 1234, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 799, in invoke
    await super().invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 1234, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 790, in invoke
    await self.prepare(ctx)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 471, in prepare
    await self._parse_arguments(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 684, in _parse_arguments
    transformed = await self.transform(ctx, param)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 526, in transform
    return await self.do_conversion(ctx, converter, argument, param)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 497, in do_conversion
    return await super().do_conversion(ctx, converter, argument, param)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 479, in do_conversion
    return await self._actual_conversion(ctx, converter, argument, param)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 438, in _actual_conversion
    raise ConversionError(converter, exc) from exc
discord.ext.commands.errors.ConversionError: (<class 'translate.converters.ChannelUserRole'>, AttributeError("'NoneType' object has no attribute 'get_channel'"))
```

Slightly different traceback for attempting to add a word (a role name on another server) 
```
﻿[2020-06-03 15:22:21] [ERROR] red: ConversionError
Traceback (most recent call last):
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 425, in _actual_conversion
    ret = await instance.convert(ctx, argument)
  File "C:\joker\cogs\CogManager\cogs\translate\converters.py", line 38, in convert
    result = discord.utils.get(guild.text_channels, name=argument)
AttributeError: 'NoneType' object has no attribute 'text_channels'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\bot.py", line 892, in invoke
    await ctx.command.invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 799, in invoke
    await super().invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 1234, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 799, in invoke
    await super().invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 1234, in invoke
    await ctx.invoked_subcommand.invoke(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 790, in invoke
    await self.prepare(ctx)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 471, in prepare
    await self._parse_arguments(ctx)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 684, in _parse_arguments
    transformed = await self.transform(ctx, param)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 526, in transform
    return await self.do_conversion(ctx, converter, argument, param)
  File "c:\envs\3.3.x\lib\site-packages\redbot\core\commands\commands.py", line 497, in do_conversion
    return await super().do_conversion(ctx, converter, argument, param)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 479, in do_conversion
    return await self._actual_conversion(ctx, converter, argument, param)
  File "c:\envs\3.3.x\lib\site-packages\discord\ext\commands\core.py", line 438, in _actual_conversion
    raise ConversionError(converter, exc) from exc
discord.ext.commands.errors.ConversionError: (<class 'translate.converters.ChannelUserRole'>, AttributeError("'NoneType' object has no attribute 'text_channels'"))
```

--- a/translate/translate.py
+++ b/translate/translate.py
@@ -36,7 +36,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
     """
 
     __author__ = ["Aziz", "TrustyJAID"]
-    __version__ = "2.3.0"
+    __version__ = "2.3.1"
 
     def __init__(self, bot):
         self.bot = bot
@@ -184,6 +184,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @translateset.group(aliases=["blocklist"])
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def blacklist(self, ctx: commands.Context) -> None:
         """
         Set blacklist options for translations
@@ -194,6 +195,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @translateset.group(aliases=["allowlist"])
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def whitelist(self, ctx: commands.Context) -> None:
         """
         Set whitelist options for translations
@@ -204,6 +206,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @whitelist.command(name="add")
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def whitelist_add(
         self, ctx: commands.Context, *channel_user_role: ChannelUserRole
     ) -> None:
@@ -225,6 +228,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @whitelist.command(name="remove", aliases=["rem", "del"])
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def whitelist_remove(
         self, ctx: commands.Context, *channel_user_role: ChannelUserRole
     ) -> None:
@@ -249,6 +253,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @whitelist.command(name="list")
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def whitelist_list(self, ctx: commands.Context) -> None:
         """
         List Channels, Users, and Roles in the servers translation whitelist.
@@ -266,6 +271,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @blacklist.command(name="add")
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def blacklist_add(
         self, ctx: commands.Context, *channel_user_role: ChannelUserRole
     ) -> None:
@@ -287,6 +293,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @blacklist.command(name="remove", aliases=["rem", "del"])
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def blacklist_remove(
         self, ctx: commands.Context, *channel_user_role: ChannelUserRole
     ) -> None:
@@ -311,6 +318,7 @@ class Translate(GoogleTranslateAPI, commands.Cog):
 
     @blacklist.command(name="list")
     @checks.mod_or_permissions(manage_messages=True)
+    @commands.guild_only()
     async def blacklist_list(self, ctx: commands.Context) -> None:
         """
         List Channels, Users, and Roles in the servers translation blacklist.

